apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java

```diff
@@ -37,94 +38,126 @@
-      consumeIfNotNull(
-          mbiMap.get(beneId).getMidName(), s -> payerBuilder.setBeneMidInit(s.substring(0, 1)));
-      consumeIf(
-          data.get(mbiMap.get(beneId).getGender()).orElse(null),
-          NumberUtils::isDigits,
-          value -> {
-            // RIF mappings are  0 - unknown, 1 - male, 2 - female
-            // RDA mappings are -1 - unrecognized, 0 - female, 1 - male, 2 - unknown
-            int enumValue = Integer.parseInt(value);
-            enumValue = (enumValue >= 0 && enumValue <= 2) ? 2 - enumValue : -1;
-            payerBuilder.setBeneSexEnumValue(enumValue);
-          });
-      consumeIfNotNull(mbiMap.get(beneId).getDob(), payerBuilder::setBeneDob);
-
-      claimBuilder.addFissPayers(
-          FissPayer.newBuilder().setBeneZPayer(payerBuilder.build()).build());
-
-      data.get(Fiss.ADMTG_DGNS_CD).ifPresent(claimBuilder::setAdmDiagCode);
-      consumeIf(
-          data.get(Fiss.CLM_FREQ_CD).orElse(null),
-          NumberUtils::isDigits,
-          value -> claimBuilder.setFreqCdEnumValue(Integer.parseInt(value)));
-      data.getFromType(Fiss.CLM_FRM_DT, Parser.Data.Type.DATE)
-          .ifPresent(claimBuilder::setStmtCovFromCymd);
-      consumeIf(
-          data.get(Fiss.CLM_SRVC_CLSFCTN_TYPE_CD).orElse(null),
-          NumberUtils::isDigits,
-          value -> claimBuilder.setServTypeCdEnumValue(Integer.parseInt(value)));
-      data.getFromType(Fiss.CLM_THRU_DT, Parser.Data.Type.DATE)
-          .ifPresent(claimBuilder::setStmtCovToCymd);
-      data.get(Fiss.CLM_TOT_CHRG_AMT).ifPresent(claimBuilder::setTotalChargeAmount);
-      consumeIf(
-          data.get(Fiss.CLM_FAC_TYPE_CD).orElse(null),
-          NumberUtils::isDigits,
-          value -> claimBuilder.setLobCdEnumValue(Integer.parseInt(value)));
-      data.get(Fiss.ORG_NPI_NUM).ifPresent(claimBuilder::setNpiNumber);
-      data.get(Fiss.PRNCPAL_DGNS_CD).ifPresent(claimBuilder::setPrincipleDiag);
-      data.get(Fiss.PRVDR_NUM)
-          .ifPresent(value -> claimBuilder.setMedaProv6(String.format("%.6s", value)));
-
-      addDiagCodes(claimBuilder, data);
-      addProcCodes(claimBuilder, data);
-
-      return FissClaimChange.newBuilder()
-          .setTimestamp(Timestamp.newBuilder().setSeconds(Instant.now().getEpochSecond()).build())
-          .setSeq(sequenceNumber.inc())
-          .setClaim(claimBuilder.build())
-          .setChangeType(ChangeType.CHANGE_TYPE_UPDATE)
-          .build();
+  public Optional<MessageOrBuilder> transform(
+      WrappedCounter sequenceNumber, Parser.Data<String> data, WrappedMessage message) {
+    FissClaimChange claimToReturn;
+
+    int lineNumber = getLineNumber(data, Fiss.CLM_LINE_NUM);
+
+    if (message.getMessage() instanceof FissClaimChange) {
+      // There is an existing claim from a previous run
+      FissClaimChange storedClaim = (FissClaimChange) message.getMessage();
+
+      if (message.getLineNumber() == (lineNumber - 1)) {
```

Comment:
For consistency, the FISS transformer is checking for additional line items, but since they aren't used yet, we're not actually doing anything with them
https://github.com/CMSgov/beneficiary-fhir-data/blob/a9a4dcd0b1b143dae77763277553feca8d8d087a/apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java#L79-L85